### DISCUSSION TITLE
明细表如何限制冻结列的最大宽度

### DISCUSSION THREAD

**Discussion by flyfly6** [antvis/S2#1782] [Q&A] [answered]
明细如果设置了比较多的冻结列，可以给冻结列添加横向滚动条吗？
https://s2.antv.vision/zh/examples/layout/custom#custom-row-col-cell-width
类似官网示例中透视表行头的这种效果，行头宽度超过表格宽度的一半且整体有横向滚动条的时候，
调整行头宽度为表格宽度的一半，并给行头添加单独的滚动条
明细表可以实现这种效果吗？

**Comment by lijinke666** [comment, marked as answer]
目前不支持, 可以提一个 feature request 的 issue
